### ISSUE TITLE
Readme file too long for Azure Artifacts

### ISSUE THREAD

**Issue by arturosevilla** [pemistahl/lingua-py#209]
I get the following error when I'm trying to setup through Azure Artifacts a project with `linguage-language-detector` as a dependency: `The package description exceeds the maximum length of 349608 bytes`

The reason, as stated, is that the package description is huge (the entire README.md file is part of the package description through the readme.

Is there a way to only have a summary of such README.md in to the package description?

**Comment by pemistahl** [comment]
Hi Arturo, thanks for reaching out to me.

> Is there a way to only have a summary of such README.md in to the package description?

I don't know whether Azure Artifacts allows to exclude or shrink the README. Can't you simply remove the README from the artifact on Azure? The problem probably occurs because of the large accuracy table I've included in the README. I could put the table in a separate file but actually I want to avoid that so that the table is available on PyPI as well.

**Comment by arturosevilla** [comment]
As far as I know, you cannot modify packages coming from upstream (PyPI). The scenario is that if we have a private repo (setup with `extra-index-url` in pip.ini), and there are some artifacts/packages that are dependent on `lingua-language-detector` then `pip` will try to restore the packages through Azure Artifacts. However, because of the aforementioned error, it will fail when being retrieved from upstream as the description is too big for Artifacts. 

Also, setting up both`extra-index-url` and/or `index-url` does not guarantee any order: https://stackoverflow.com/questions/67253141/python-pip-priority-order-with-index-url-and-extra-index-url

Other packages have had this issue:
- https://github.com/pydantic/pydantic/issues/8125
- https://github.com/Azure/azure-cli/issues/18383

**Comment by pemistahl** [comment]
Ok, I will put the table in a separate file when I prepare the next release. But I won't make a new release just for the sake of the readme. So you will have to wait until the next regular release. I hope you understand.

**Comment by arturosevilla** [comment]
Of course. I just wanted it bring this to attention. In fact, I believe Azure Artifacts is going against the standard, but I do understand they want to impose certain limits.

**Comment by diwakar-vsingh** [comment]
Hi Peter, is there any update on this as I'm running into the exact same error that Arturo was running into?

**Comment by pemistahl** [comment]
@diwakar-vsingh Not yet, unfortunately. I've been busy with my daily job and family life, so I could not continue the development of library yet. There will be updates for sure, but I cannot tell you when right now.

**Comment by carloshm91** [comment]
Hello @pemistahl ! There is any update on this issue? Or maybe some workaround about this?

**Comment by pemistahl** [comment]
@carloshm91 I will release 2.1.0 soon where this will be fixed. In the meantime you can use 1.4.0 which is pure Python so somewhat slower. 2.1.0 will be on par with 1.4.0 with regard to features and bug fixes.

**Comment by carloshm91** [comment]
that is great! waiting for the release

**Comment by carloshm91** [comment]
hello @pemistahl how is going? any update on this?

**Comment by pemistahl** [comment]
I'm going to release 2.1.0 next week.
